appsec-kit-v8/src/main/java/com/vaadin/appsec/v8/ui/content/VulnerabilitiesTab.java

```diff
@@ -109,6 +109,8 @@ private void buildGrid() {
                 .setCaption("Vulnerability name or identifier");
         grid.addColumn(VulnerabilityDTO::getDependency)
                 .setCaption("Dependency");
+        grid.addColumn(v -> v.getDependency().getVersion())
+                .setCaption("Version");
```

Comment:
Should we add a version column in the v7 version too?

Reply:
Yes, we should as part of #49 but I need to understand how to do so in V7 😅

Reply:
I removed that from V8, let's add it later properly as part of #49